### ISSUE TITLE
Doesn't work on Expo

### ISSUE THREAD

**Issue by KingAxanthic** [Aashu-Dubey/React-Native-UI-Templates#2]
![image](https://user-images.githubusercontent.com/106954969/193423551-6500e887-769e-4e34-96b6-e84ca90326bf.png)

**Comment by firstChairCoder** [comment]
The project is built on bare React Native, so that would be the result...

**Comment by Aashu-Dubey** [comment]
Yes, I'm aware. This project is quite old now, so is not compatible with latest expo versions. I need to upgrade the project to make it compatible, so for now you have to go with bare react-native installation flow.

I had put the upgradation on hold as I was thinking to first also try the project on other supported platforms (Web, Windows and MacOS), which I will soon do.

**Comment by dhatGuy** [comment]
Any update?
